Common/Inc/CommonDataTypes.hpp

```diff
@@ -1,21 +1,35 @@
-//
-// Created by Gordon Fountain 2022-11-30.
-//
+/** CommonDataTypes.hpp
+ * Contains a bunch of common datatypes!
+ * 
+ * Created by: Gordon Fountain 2022-11-30
+ * Updated by: Anthony Luo
+ * Last update: 2024-04-18
+*/
 
 #ifndef ZPSW3_COMMON_DATATYPES_HPP
 #define ZPSW3_COMMON_DATATYPES_HPP
 
 #include <stdint.h>
 
+/************************************************
+ * Sensor/Error structs (can be either or. tbd ig)
+ ************************************************/
+typedef enum SensorErrorCode {
+    SENSOR_SUCCESS = 0,
+    SENSOR_FAIL = -1,
+    SENSOR_BUSY = 1,
+} SensorErrorCode;
```

Comment:
This enum name doesn't follow the style guide: https://uwarg-docs.atlassian.net/wiki/spaces/ZP/pages/2268692486/Style+Standards